spec/integration/user_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe 'UserShows', type: :system do
+  subject { User.new(name: 'John', posts_counter: 30, photo: 'https://randomuser.me/api/portraits/men/70.jpg', bio: 'Teacher from Poland.') }
+
```

Comment:
- [ ] As mentioned by the previous reviewer, You still have a lot of failed tests because you created a column called `post_counter` and you used here `posts_counter` instead and that's what causing the problem. Please fix that 

![image](https://user-images.githubusercontent.com/68796232/216047511-4bfb4c70-f7ad-4f24-a12d-7f20ece59df2.png)